tests/conftest.py

```diff
@@ -373,8 +373,9 @@ def pytest_runtest_teardown(item: pytest.Item) -> None:
 
     Dumps the current UI test report HTML.
     """
-    if item.session.config.getoption("ui"):
-        testreport.index()
+    # Not generating any reports to make tests quicker.
```

Comment:
Reverted in [f889c12](https://github.com/trezor/trezor-firmware/pull/2580/commits/f889c12b2fbcaf485a310965812329b998705b7a) - but I am not very happy about it, as it quite significantly slows down the `device tests`, because of generating the `index.html` after each test-case (probably even after skipped ones). It is really visible how the test-suite slows down at the end, as it needs to generate bigger and bigger report.

I think we could have some flag to disable this generation and pass it in the `make test_emu_*` (if we want the possibility of creating the report gradually in manual `pytest` callings).

Reply:
Opened https://github.com/trezor/trezor-firmware/issues/2640